core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -761,6 +769,15 @@
         "alluxio.user.ufs.file.reader.class";
     public static final String USER_UFS_FILE_WRITER_CLASS =
         "alluxio.user.ufs.file.writer.class";
+    public static final String USER_UFS_FILE_READ_LOCATION_POLICY =
+        "alluxio.user.ufs.file.read.location.policy";
+    public static final String USER_UFS_FILE_READ_DETERMINISTIC_HASH_POLICY_NUM_SHARDS =
+        "alluxio.user.ufs.file.read.deterministic.hash.policy.num.shards";
```

Comment:
suggestion: how about `alluxio.user.ufs.file.read.location.policy.deterministic.hash.shards` or
`alluxio.user.ufs.block.location.policy.deterministic.hash.shards` if you decide to move the class to block package as suggested by jiri

also
`alluxio.user.ufs.file.read.location.policy` can be `alluxio.user.ufs.block.location.policy` if you move the class to where jiri suggested.